.github/workflows/build-jobs.yaml

```diff
@@ -173,10 +186,55 @@ jobs:
           cd ../build_relwithdebinfo
           ninja ytserver-yql-agent
 
+      - name: Build query tracker (ya make, relwithdebinfo)
+        if: ${{ inputs.build-query-tracker-ya-make }}
+        run: |
+          mkdir -p ../build_qt_ya_make
+          cd ../build_qt_ya_make
+          rm -rf *
+
+          ../ytsaurus/ytsaurus/ya make -T --build=relwithdebinfo  --output=../ya_build ../ytsaurus/ytsaurus/yt/yql/agent/bin
```

Comment:
It takes about 3 hours to complete a single build. So, in case of nightly builds, I suggest to build only the RelWithDebInfo version